### ISSUE TITLE
A11y_AzureDataStudio_Filter_ScreenReader: Orca does not announce the control type for the checkboxes present in the "Filter" section.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#16022]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11yMAS;#A11yTCS;A11ySev2;#SH-AzureDataStudio-Linux-Jul2021;#SQL Azure Data Studio;#Linux;#DesktopApp;#WCAG4.1.2;#Orca;#AINotInScope;#FTP;

### Environment Details:
Azure Data Studio Version: 1.31.0-insiders
OS Name: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
Screen Reader: Orca

### Steps to Reproduce:

1. Launch Azure Data Studio
2. Connect to any SQL server and write a query
3. TAB till results grid and press F3 so that the filters pop-up will open
4. Start Orca screen reader and then TAB till "Filter" section.
5. Using DOWN arrow key navigate to first checkbox.
6. Observe the announcement by screen reader

### Actual:
The screen reader only announces the name of the checkbox but it does not announce the control type as well as whether the control is selected or not. 

### Expected:
The screen reader should announce the name, control type as well as the state of the control. For example so here it should announce it like "test checkbox non-selected" 

User Impact:
Screen reader users won't be able to understand the purpose of the control.

**Comment by alanrenmsft** [comment]
![image](https://user-images.githubusercontent.com/13777222/125144105-ae2e7900-e0d1-11eb-9b07-b0a5336eaef8.png)
this is the html, we are using standard html here, if Orca decides not to read the control type, I don't think there is anything we can do from our side.

**Comment by 12-shweta** [comment]
After discussion with SME this seem to just be the differences in how Orca announces these elements

**Comment by manaswita-chougule** [comment]
#Regressed:07-18-23; #Closed;
This is an Orca screen reader behavior that it will not announce the control type(checkbox) and it's state. Hence closing the bug as by-design.
